packages/webapp/src/components/Form/Input/index.jsx

```diff
@@ -14,6 +14,8 @@ import { get } from 'react-hook-form';
 import i18n from '../../../locales/i18n';
 
 const Input = ({
+  openCalendar,
+  autoFocus,
```

Comment:
@MwayaB 
This shouldn't be necessary anymore either if you remember the way you had it originally it was getting passed down through {...props} the way you had it before.